packages/server/lib/request.coffee

```diff
@@ -66,8 +66,12 @@ newCookieJar = ->
 
     setCookie: (cookieOrStr, uri, options) ->
       ## store the original URL this cookie was set on
-      cookie = j.setCookieSync(cookieOrStr, uri, options)
-      cookie.url = uri
+      if cookie = j.setCookieSync(cookieOrStr, uri, options)
```

Comment:
hehe, have you seen my code in a paranoid style where arguments and results are constantly asserted? this is why